psh/test-ls-rootfs.py

```diff
@@ -65,6 +65,17 @@ def assert_ls_t(p):
     psh.assert_cmd(p, f'ls -t {ROOT_TEST_DIR}', expected_pattern, msg, is_regex=True)
 
 
+def assert_ls_S(p):
+    expected_pattern = r'.*?psh' + SEPARATOR_PATTERN + r'.*?empty_file' + SEPARATOR_PATTERN + r'.*?'
+    expected_pattern = expected_pattern + psh.EOL
+
+    psh.assert_cmd(p, 'touch /bin/empty_file', '', 'Wrong output when creating empty file!')
+
+    msg = 'Wrong output, when calling `ls -1S`'
+    # TODO: use only newly created files, when it will be possible to write content to a file
+    psh.assert_cmd(p, 'ls -1S /bin', expected_pattern, msg, is_regex=True)
```

Comment:
```suggestion
    # TODO: use only newly created files, when it will be possible to write content to a file
    psh.assert_cmd(
        pexpect_proc=p, 
        cmd='ls -1S /bin', 
        expected=expected_pattern, 
        msg='Wrong output, when calling `ls -1S`',
        is_regex=True)
```

does this fit under "120 char max python line` rule?

Reply:
Yes, maybe it's also a better solution, thanks.